cmd/command.go

```diff
@@ -64,6 +64,24 @@ func (app *App) Bind(cmd *cobra.Command) {
 		"Listen port for the HTTP server. (ENV:MERGER_PORT)")
 	app.viper.BindPFlag("port", cmd.PersistentFlags().Lookup("listen-port"))
 
+	cmd.PersistentFlags().Int(
+		"exporters-timeout", 10,
+		"HTTP client timeout for connecting to exporters. (ENV:MERGER_EXPORTERS_TIMEOUT)")
+	app.viper.BindPFlag("exporters-timeout", cmd.PersistentFlags().Lookup("exporters-timeout"))
+
+	cmd.PersistentFlags().String(
+		"log-level", "error",
+		"Log level (possible values: debug, info, warning, error, fatal, panic). (ENV:MERGER_LOG_LEVEL)")
+
+	app.viper.BindPFlag("log-level", cmd.PersistentFlags().Lookup("log-level"))
+
+	loglevel, err := log.ParseLevel(app.viper.GetString("log-level"))
```

Comment:
Viper has not parsed the command line flags at this point. You need to do this inside of a `*Run` command (like `PersistentPreRun`.

Reply:
Thanks for the hint, fixed! Now it works.